customedittextoutlinedborder/src/main/java/com/simform/customcomponent/SSCustomEdittextOutlinedBorder.kt

```diff
@@ -122,6 +122,12 @@ class SSCustomEdittextOutlinedBorder @JvmOverloads constructor(context: Context,
         }
     }
 
+    fun getCLickListener(onclick:() -> Unit){
```

Comment:
Please rename it to getClickListener and format the code.

Reply:
Resolved.